test/common/module_api.test.lua

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env tarantool
+
+local socket = require('socket')
+local tap = require('tap')
+package.cpath = './?.so;' .. package.cpath
+local memcached = require('memcached')
+local test = tap.test('memcached module api')
+
+local function is_port_open(port)
+    local sock, _ = socket.tcp_connect('127.0.0.1', port)
+    if not sock then
+        return false
+    end
+    return sock:sysconnect(0, port) and true or false
+end
+
+if type(box.cfg) == 'function' then
+    box.cfg{
+        wal_mode = 'none',
+        memtx_memory = 100 * 1024 * 1024,
+    }
+    box.schema.user.grant('guest', 'read,write,execute', 'universe')
+end
+
+test:plan(15)
+
+-- memcached.server: module is initialized, no instances
+
+test:istable(memcached.server, 'type of memcached.server is a table')
+test:is(#memcached.server, 0, 'memcached.server is empty')
```

Comment:
I don't get the point of testing this.

Reply:
`memcached.server` is empty when no started instances.

Reply:
But then why is it also empty on lines 63-64 (you missed the comment there apparently)?

AFAIC, you cannot even use the length operator on this table, as it is not an array:
https://github.com/tarantool/memcached/blob/5aaac8383ee9b04ecaac701c289ac3712ba48f8c/memcached/init.lua#L379-L382